source/io/module_httpClient.js

```diff
@@ -591,6 +592,19 @@
                 Module.eggShell.setOccurrenceAsync(occurrencePointer);
             });
         };
+
+        Module.httpClient.jsHttpClientSetCredentials = function (handle, isWithCredentials, errorStatusPointer, errorCodePointer, errorSourcePointer) {
+            var errorStatus = Module.eggShell.dataReadBoolean(errorStatusPointer);
+            if (errorStatus) {
+                return;
+            }
+
+            var httpClient = findhttpClientOrWriteError(handle, 'LabVIEWHTTPClient:Credentials', errorStatusPointer, errorCodePointer, errorSourcePointer);
+            if (httpClient === undefined) {
+                return;
+            }
+            httpClient.withCredentials = (isWithCredentials !== 0);
```

Comment:
It looks like this is just tacking on the withCredentials property to this object instance.

Instead it should be managed by the HttpClient class.

in the HttpClient constructor it should set this._withCredntials to false
On the prototype a setWithCredentials function should be added (As withCredentials is only used internally by the HTTPClient a getWithCredentials is not necessary so do not add that)

On this line we then invoke httpClient.setWithCredentials() instead of assigning directly to a property